src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -434,6 +395,57 @@ public void canOverrideRenewalWhenLoanReachedRenewalLimitAndDueDateIsNotSpecifie
         OVERRIDE_COMMENT, null)
         .getJson();
 
+    verifyRenewedLoan(smallAngryPlanet, jessica, renewedLoan);
+
+    assertThat("renewal count should be incremented",
+      renewedLoan.getInteger("renewalCount"), is(2));
+
+    DateTime expectedDueDate = loanDate.plusWeeks(3);
+    assertThat("due date should be 3 weeks later",
+      renewedLoan.getString("dueDate"),
+      isEquivalentTo(expectedDueDate));
+  }
+
+  @Test
+  public void canOverrideRenewalWhenItemIsDeclaredLost() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+    LoanPolicyBuilder limitedRenewalsPolicy = new LoanPolicyBuilder()
+      .withName("Limited Renewals policy")
+      .rolling(Period.weeks(1));
+
+    use(limitedRenewalsPolicy);
+
+    final DateTime loanDate = DateTime.now(DateTimeZone.UTC).minusWeeks(1);
+
+    final JsonObject loanJson = loansFixture.checkOutByBarcode(smallAngryPlanet,
+      usersFixture.jessica(), loanDate).getJson();
+
+    declareItemLost(loanJson);
+
+    loansFixture.attemptRenewal(422, smallAngryPlanet, jessica);
+
+    final JsonObject renewedLoan =
+      loansFixture.overrideRenewalByBarcode(smallAngryPlanet, jessica,
+        OVERRIDE_COMMENT, null).getJson();
+
+    verifyRenewedLoan(smallAngryPlanet, jessica, renewedLoan);
+
+    assertThat("renewal count should be incremented",
+      renewedLoan.getInteger("renewalCount"), is(1));
+
+    assertThat("item status should be changed",
+      renewedLoan.getJsonObject("item").getJsonObject("status").getString("name"),
+      is(ItemStatus.CHECKED_OUT.getValue()));
+
+    DateTime expectedDueDate = loanDate.plusWeeks(2);
+    assertThat("due date should be 2 weeks later",
+      renewedLoan.getString("dueDate"),
+      isEquivalentTo(expectedDueDate));
+  }
+
+  private void verifyRenewedLoan(IndividualResource smallAngryPlanet,
```

Comment:
Please can these private methods be moved till after the tests, so that they distract less from the tests themselves

Reply:
Done